codecov.yml

```diff
@@ -1,2 +0,0 @@
-fixes:
-  - '::offchain-modules/'
```

Comment:
It will break codecov show for offchain coverage test.
Should figure out a way to support contracts and offchain both.